src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2332,11 +2332,19 @@ private void embedGenericHandle(ref CORINFO_RESOLVED_TOKEN pResolvedToken, bool
 
                 if (!runtimeLookup)
                 {
-                    throw new NotImplementedException("LDTOKEN Method");
+                    if (pResolvedToken.tokenType == CorInfoTokenKind.CORINFO_TOKENKIND_Ldtoken)
+                        pResult.lookup.constLookup.handle = (CORINFO_GENERIC_STRUCT_*)ObjectToHandle(_compilation.NodeFactory.RuntimeMethodHandle(md));
+                    else
+                        throw new NotImplementedException();
                 }
```

Comment:
Not sure what other token types could appear here, so i'm making it throw for now.
I believe the CORINFO_TOKENKIND_Method shouldn't really appear for the !runtimeLookup case, right?